docs/markdown/snippets/response_file.md

```diff
@@ -0,0 +1,5 @@
+## New `response_file` function
+
+This function allows to convert a list of arguments
+into a response file, to use with `custom_target` or
+other functions calling a custom executable.
```

Comment:
There is probably a better way to phrase this last part, but not sure.

Reply:
I tried something else...